action.yml

```diff
@@ -12,7 +12,8 @@ inputs:
     required: true
   tf_version:
     description: 'Terraform version to install.'
-    required: true
+  tofu_version:
```

Comment:
I think `tofu_version` should be added to list of arguments

https://github.com/gruntwork-io/terragrunt-action?tab=readme-ov-file#inputs